workflows.py

```diff
@@ -0,0 +1,64 @@
+# @@@SNIPSTART python-money-transfer-project-template-workflows
+from datetime import timedelta
+
+from temporalio import workflow
+from temporalio.common import RetryPolicy
+from temporalio.exceptions import ApplicationError
+
+with workflow.unsafe.imports_passed_through():
+    from activities import deposit, refund, withdraw
+    from shared import PaymentDetails
+
+
+@workflow.defn
+class MoneyTransfer:
+    @workflow.run
+    async def run(self, payment_details: PaymentDetails) -> str:
+        retry_policy = RetryPolicy(
+            maximum_attempts=3,
+            maximum_interval=timedelta(seconds=2),
+            non_retryable_error_types=["InvalidAccountError", "InsufficientFundsError"],
+        )
+
+        # Withdraw money
+        withdraw_output = await workflow.execute_activity(
+            withdraw,
+            payment_details,
+            start_to_close_timeout=timedelta(seconds=5),
+            retry_policy=retry_policy,
+        )
+
+        # Deposit money
+        try:
+            deposit_output = await workflow.execute_activity(
+                deposit,
+                payment_details,
+                start_to_close_timeout=timedelta(seconds=5),
+                retry_policy=retry_policy,
+            )
+        except ApplicationError as deposit_err:
```

Comment:
```suggestion
        except ActivityError as deposit_err:
```

I didn't think `execute_activity` would ever raise `ApplicationError` directly (but maybe there's some kind of bug or thing I am unaware of). Do you have a test confirming this catch clause is executed?

Reply:
Replaced `ApplicationError` with `ActivityError`.